pkg/util/node/node_test.go

```diff
@@ -259,3 +263,106 @@ func TestGetHostname(t *testing.T) {
 
 	}
 }
+
+func TestGetNodeIP(t *testing.T) {
```

Comment:
this function is a wrapper on `GetNodeHostIP` , you can reuse the same testcases than `TestGetNodeHostIPs`